plugin/src/main/java/org/wildfly/swarm/plugin/maven/PackageMojo.java

```diff
@@ -69,6 +69,12 @@
     @Parameter(alias = "hollow", defaultValue = "false", property = "swarm.hollow")
     protected boolean hollow;
 
+    /**
+     * Flag to skip packaging entirely.
+     */
+    @Parameter(alias = "skip", defaultValue = "false", property = "swarm.package.skip")
```

Comment:
Actually I just blindly copied this from `hollow`. Does this really influence the CLI?